djoser/utils.py

```diff
@@ -21,13 +20,15 @@ def decode_uid(pk):
 
 
 def login_user(request, user):
-    token, _ = authtoken.models.Token.objects.get_or_create(user=user)
+    Token = import_string(settings.get('TOKEN_MODEL'))
```

Comment:
I was thinking the same, just waiting for someone to tell me how to do it... :blush: